dev-docs/source/CheckingOutTheMantidCode.rst

```diff
@@ -0,0 +1,117 @@
+.. _CheckingOutTheMantidCode:
```

Comment:
Should this file be here? This information is duplicated in `GettingStarted.rst`.

Reply:
That's a leftover from someone else, I'll delete that.